### ISSUE TITLE
Python Blob cannot store list of Strings

### ISSUE THREAD

**Issue by ActiveBrainAtlas** [datajoint/datajoint-python#520]
When storing a list of data inside a blob, string datatypes are not functional in Python. 

When attempting to store a string list into a blob an error like the following will be thrown:
```
~/code/Python_3/lib/python3.7/site-packages/datajoint/blob.py in pack_array(array)
    252         array, imaginary = np.real(array), np.imag(array)
    253 
--> 254     type_number = rev_class_id[array.dtype]
    255 
    256     if dtype_list[type_number] is None:

KeyError: dtype('<U1')
```

**Comment by eywalker** [comment]
Let's add native support for Python's bytes (b), string (w), dictionary (d), list (l), tuples (t), and set (m) into mYm by adding new data types.

**Comment by dimitri-yatsenko** [comment]
Also, perhaps `dj0` should have special support Python's built-in `int` type.  Also, perhaps we could support `dtype`s that MATLAB does not support e.g. `float16` and `float128`.

**Comment by dimitri-yatsenko** [comment]
Once we release `datajoint-matlab` that can work with `dj0`, then we can always label blobs as `dj0` even when not using the new datatypes.

**Comment by FedeClaudi** [comment]
Hi, 

Is there a ETA for when this feature will be available?

**Comment by eywalker** [comment]
Hoping to have it implemented by sometime next week.

**Comment by FedeClaudi** [comment]
Hi, any progress on this issue?

**Comment by dimitri-yatsenko** [comment]
Yes, coming out in a few days.  Sorry for the delay.   Back to work after the holidays!

**Comment by dimitri-yatsenko** [comment]
Fixed with #577
